config/config.yml

```diff
@@ -1,4 +1,7 @@
 defaults: &defaults
+  email:
+    from_address: plum@princeton.edu
+    reviewer_address: cams@princeton.edu
```

Comment:
There are a few options here: a server-managed list, or a list that could be in the to/cc of a single email, or separate emails sent to each member.  A server-managed list would decouple updating the list from updating the app.